3_ProtectionProfile/PP_Config-OS.adoc

```diff
@@ -0,0 +1,333 @@
+|A.Authentication
+|A.User
+
+|===
+
+=== Consistency of Objectives
+
+The objectives for the biometric system and its operational environment are consistent with the <<PP_OS_V4.3>> based on the following rationale:
+
+.Consistency Rationale for TOE Objectives
+[cols=".^1,.^1",options="header"]
+|===
+|PP-Module TOE Objectives	
+|Consistency Rationale
+
+|O.BIO_Verification	
+.3+|These TOE Objectives are specific subsets of the O.INTEGRITY objective in the <<PP_OS_V4.3>>. 
+|O.Enrol
+
+|O.Protection	
+|This TOE Objective is specific subset of the O.PROTECTED_STORAGE objective in the <<PP_OS_V4.3>>.
+
+|===
+
+.Consistency Rationale for Environmental Objectives
+[cols=".^1,.^1",options="header"]
+|===
+|PP-Module Environmental Objectives	
+|Consistency Rationale
+
+|OE.Alternative	
+.4+|All Environmental Objectives levied on the operational environment of biometric system (i.e. The computer) are consistent with security requirements in the <<PP_OS_V4.3>>. 
+|OE.Authentication
+|OE.Protection
+|OE.User
+
+|===
+
+=== Consistency of Requirements
+
+The Biometric System (i.e. TSF in the <<BIOPP-Module>>) is comprised of biometric capture sensors and firmware/software that provide functions described in the <<BIOPP-Module>> TOE design. The Biometric System may be integral to the computer or an external (self-contained) device that has been linked/paired with the computer. The Biometric System is invoked by the computer as defined in the <<PP_OS_V4.3>> when user’s biometric characteristics is presented to the sensor. The Biometric System creates and stores the template or compares the features with the stored template and returns the verification outcome to the computer.
+
+The <<BIOPP-Module>> assumes that the computer satisfies SFRs defined in the <<PP_OS_V4.3>> so that the Biometric System can work as specified in the <<BIOPP-Module>>. This section explains which SFRs in the <<PP_OS_V4.3>> are directly relevant to the Biometric System security functionality.
+
+The following rationale identifies several SFRs from <<PP_OS_V4.3>> that are needed to support Biometric System functionality and explains why the unions of SFRs in the <<PP_OS_V4.3>> and <<BIOPP-Module>> do not lead to a contradiction.
+
+==== Relation among SFRs/OEs in the PP_OS_V4.3 and BIOPP-Module
+The relation between SFRs defined in the <<PP_OS_V4.3>> and SFRs and OEs in the <<BIOPP-Module>> is described below for each security functionality. *Bold SFRs* are those SFRs defined in the <<BIOPP-Module>> for the Biometric System and _italicized SFRs_ are those defined in <<PP_OS_V4.3>> for the computer.
+
+===== Authentication Factors
+The authentication factors defined in the <<PP_OS_V4.3>> are Non-Biometric Authentication Factors as defined in the <<BIOPP-Module>>. The computer shall implement an authentication factor as required by the _FIA_UAU.5.1._ These authentication factors are used as an alternative authentication mechanism when the user is rejected by the biometric verification.
```

Comment:
computer

Replace
"The _computer_ shall implement an authentication factor as required by the _FIA_UAU.5.1._ These authentication factors are used as an alternative authentication mechanism when the user is rejected by the biometric verification"
with:
"The **TOE** shall implement an authentication factor as required by the _FIA_UAU.5.1._ These authentication factors are used as an alternative authentication mechanism when the user is rejected by the biometric verification"